frontend/src/__tests__/cypress/cypress/pages/userManagement.ts

```diff
@@ -0,0 +1,68 @@
+import { Contextual } from './components/Contextual';
+
+class GroupSettingSection extends Contextual<HTMLElement> {
+  shouldHaveAdministratorGroupInfo() {
+    this.find().findByTestId('data-science-administrator-info');
+    return this;
+  }
+
+  clearMultiChipItem() {
+    this.find().findByRole('button', { name: 'Clear all' }).click();
+  }
+
+  findMultiSelectGroupInput() {
+    return this.find().find('input');
+  }
+
+  findMultiGroupOptions(name: string) {
+    return this.find().findByTestId('multi-group-selection').findByRole('option', { name });
+  }
+
+  private findChipGroup() {
+    return this.find().findByRole('list', { name: 'Chip group category' });
+  }
+
+  findChipItem() {
+    return this.findChipGroup().find('li');
+  }
```

Comment:
This would be more useful as a parameterized function that attempts to locate the specific item by name. That way `exists` and `not.exists` checks can be chained.